seahorn/jobs_unsafe/mem_realloc_unsafe/aws_mem_realloc_unsafe_harness.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * 
+ */
+
+#include <seahorn/seahorn.h>
+#include <aws/common/allocator.h>
+#include "proof_allocators.h"
+#include "nondet.h"
+
+int main() {
+
+    struct aws_allocator *allocator = nd_bool() ? sea_allocator() : sea_allocator_with_realloc();
+
+    void *data = NULL;
+
+    size_t old_size = nd_size_t();
+    assume(old_size <= UINT8_MAX);
```

Comment:
is this assume really needed?

Reply:
Not necessary. The size could be unbounded.